### PR TITLE
Add LogInventory functionality.  After various inventory modifying ac…

### PR DESCRIPTION
**Pull request by reuscam**
Bot will dump contents of inventory to <playername>.json file.  Third parties can consume the file for whatever purpose.  Example of a crappy viewer (consumer) here: https://snag.gy/VBixEN.jpg

### CODE REVIEW

File: PoGo.NecroBot.Logic/Utils/Statistics.cs [hunk 1]
```diff
@@ -104,6 +105,21 @@ public void SetUsername(GetPlayerResponse profile)
...
+        public static async System.Threading.Tasks.Task LogInventory(PoGo.NecroBot.Logic.State.ISession session)
+        {
+            if (session.LogicSettings.LogInventory)
+            {
+                GetInventoryResponse inventory = await session.Inventory.GetCachedInventory();
+                var now = DateTime.UtcNow;
+
+                if (now.Ticks > _lastRefresh.AddSeconds(session.LogicSettings.LogInventoryDelaySeconds).Ticks)
+                {
+                    System.IO.File.WriteAllText(_playerName + ".json", inventory.InventoryDelta.InventoryItems.ToString());
```
Comment (Ilses) [inline PoGo.NecroBot.Logic/Utils/Statistics.cs]: You probably need a path to write to

**Comment by Ilses** [comment]
You probably want something like:
```c#
var inventoryFile = System.IO.File.WriteAllText(Path.Combine(session.LogicSettings.ProfilePath, "temp", _playerName + ".json");`
in there. Look at UseIncubatorsTask.cs for reference.

**Comment by Ilses** [comment]
Odd, automerge appears to be complaining about a trailing whitespace error

**Comment by reuscam** [comment]
can you send the erro

**Comment by NecronomiconCoding** [comment]
This branch has conflicts that must be resolved

**Pull request closed without merging**